test/suites/standard1/networking-smoke.robot

```diff
@@ -20,13 +20,11 @@ ${HOSTNAME}                 hello-microshift.cluster.local
 *** Test Cases ***
 Router Smoke Test
     [Documentation]    Run a router smoke test
-    [Tags]    robot:exclude
     [Setup]    Run Keywords
-    ...    Restart Greenboot And Wait For Success
```

Comment:
So this assumes the VM is already healthy when the test starts?

Reply:
Instead of waiting for the whole system it just waits for the components it needs to test, that is apiserver and router.